pkg/js/helpers.js

```diff
@@ -445,69 +466,66 @@ function addRecord(d,type,name,target,mods) {
 
 //ip conversion functions from http://stackoverflow.com/a/8105740/121660
 // via http://javascript.about.com/library/blipconvert.htm
-function IP(dot)
-{
+function IP(dot) {
     var d = dot.split('.');
-    return ((((((+d[0])*256)+(+d[1]))*256)+(+d[2]))*256)+(+d[3]);
+    // prettier-ignore
+    return ((((((+d[0]) * 256) + (+d[1])) * 256) + (+d[2])) * 256) + (+d[3]);
```

Comment:
Without prettier-ignore, this line turns to

```js
return ((+d[0] * 256 + +d[1]) * 256 + +d[2]) * 256 + +d[3];
```

I'm not sure which is better, so I put spaces around the operands in the original line instead.